### ISSUE TITLE
Sombra Bug

### ISSUE THREAD

**Issue by PomfPomfPluff** [Overwatch-1-Emulator/ow1-emulator#1597]
Sometimes when you pick Sombra you sometimes get to use Virus and new Translocator alongside your old abilities.

**Comment by arvrianmaybe** [comment]
yeah, is just an initiliaze issue im trying to find a way to fix it. I assume yours applys on new rounds cause that happened to me.

**Comment by MaxwellJung** [comment]
There's also similar issue on hog where you can sometimes use your pig open

**Comment by MaxwellJung** [comment]
I think issue's caused by asynchronous initialization

**Comment by arvrianmaybe** [comment]
> There's also similar issue on hog where you can sometimes use your pig open

I'm pretty sure this applies to every hero that overwrite there abilities such as orisa which fully overwrites her kit, doomfist doesn't apply cause the only thing that jt does is his powerblock which we still use nonetheless.
